src/ui/MessageContent/index.tsx

```diff
@@ -390,6 +445,63 @@ export default function MessageContent(props: MessageContentProps): ReactElement
           />
         )
       }
+      {
+        showFeedbackOptionsMenu && (
+          <MobileFeedbackMenu
+            hideMenu={() => {
+              setShowFeedbackOptionsMenu(false);
+            }}
+            onEditFeedback={() => {
+              setShowFeedbackOptionsMenu(false);
+              setShowFeedbackModal(true);
+            }}
+            onRemoveFeedback={async () => {
+              await message.deleteFeedback(message.myFeedback.id);
+              setSelectedFeedback(null);
+              setShowFeedbackOptionsMenu(false);
+            }}
+          />
+        )
+      }
+      {/* Feedback Modal */}
+      {
+        showFeedbackModal && (
+          <MessageFeedbackModal
+            selectedFeedback={selectedFeedback}
+            message={message}
+            onSubmit={async (comment: string) => {
+              if (selectedFeedback) {
+                if (message.myFeedback) {
+                  if (comment !== message.myFeedback.comment) {
+                    const newFeedback: Feedback = new Feedback({
+                      id: message.myFeedback.id,
+                      rating: selectedFeedback,
+                      comment,
+                    });
+                    await message.updateFeedback(newFeedback);
+                  }
```

Comment:
```suggestion
                if (!message.myFeedback) {
                  return;
                }
                if (comment !== message.myFeedback.comment) {
                  const newFeedback: Feedback = new Feedback({
                    id: message.myFeedback.id,
                    rating: selectedFeedback,
                    comment,
                  });
                  await message.updateFeedback(newFeedback);
                }
```

There're 3 nested if stmts here. How about we use the early-return pattern like the above for the better reabability? It'll reduce 1 level ;)

Reply:
oh no, if `!message.myFeedback`, we still need to do two actions:
1. submitFeedback.
2. setSelectedFeedback(null);

And setSelectedFeedback is a common action so I think its better to keep it as is. What do you think?

Reply:
Oh there's another logic below the block. Still I think we can do something. Then how about this?
```
if (!selectedFeedback) { // let's separate the first if block
  return;
}
if (message.myFeedback) {
  ...
} else {
  ...
}
setShowFeedbackModal(false);
```

Reply:
Yes that is better 👍